tests/tools/oesign/sign-and-verify.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+import sys
+import subprocess
+import argparse
+
+if __name__ == "__main__":
+
+    arg_parser = argparse.ArgumentParser(description="Invokes the oesign sign command and attempts to load the signed enclave")
+    arg_parser.add_argument('--oesign-path', default=None, type=str, help="Path to the oesign tool")
+    arg_parser.add_argument('--oesign-args', default=None, type=str, help="Additional arguments to be passed to oesign")
+    arg_parser.add_argument('--enclave-path', default=None, type=str, help="Path to the enclave binary to be signed")
+    arg_parser.add_argument('--host-path', default=None, type=str, help="Path to the enclave host app used to launch the enclave")
+
+    args = arg_parser.parse_args()
+    print(args)
+
+    if not args.enclave_path or \
+       not args.host_path or \
+       not args.oesign_args or \
+       not args.oesign_path:
+
+        arg_parser.print_help()
+        sys.exit(1)
+
+    sign_cmd = args.oesign_path + ' sign --enclave ' + args.enclave_path + ' ' + args.oesign_args
+    launch_cmd = args.host_path + ' ' + args.enclave_path + '.signed'
+
+    retval = subprocess.call(sign_cmd, shell=True)
```

Comment:
This is a new script; it runs two separate commands (oesign, then launch the signed enclave if it succeeded) as a single command, since CMake `add_test()` does not accept multiple `COMMANDS`. The alternative is to have interdependent ctests, which is undesirable since the output dependencies cannot be formally specified.

Reply:
Another approach is to use '&&' to chain the two commands. I don't have a strong preference either way.

Reply:
I started out trying `&&` but that's a bash construct, so the command needs to invoke it through `bash -c` and it gets weirdly broken when run on Windows. Also, this is a precursor to the digest signing tests which add even more shell operations to a single test, and that approach rapidly becomes bash scripting on a single line; python remains a fairly robust way to keep it readable and useful on both OS.